R/objectstore.R

```diff
@@ -27,12 +27,14 @@
 list_keys <- function(client,
                       datasource,
                       prefix = "",
-                      override = list()) {
+                      override = list(),
+                      page_size = 50) {
```

Comment:
can you regenerate the `man/*Rd` files to match this?

Reply:
`devtools::document()` should do that easily